### PR TITLE
Add rest controllers

### PR DESCRIPTION
**Pull request by randrii**

### CODE REVIEW

File: src/main/java/com/rybka/controller/BankController.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("banks")
+public class BankController {
+    @Autowired
+    private BankAggregatorService service;
```
Comment (jarc0der) [inline src/main/java/com/rybka/controller/BankController.java]: Why Autowired? Constructor is bad idea?

File: src/main/java/com/rybka/controller/BankController.java [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+    @GetMapping(path = "/{base}", produces = "application/json")
```
Comment (jarc0der) [inline src/main/java/com/rybka/controller/BankController.java]: We have RestController on the class name, as far as I remember in produces application/json by default. 
Try to remove produces.

File: src/main/java/com/rybka/controller/BankController.java (hunk 1 above)
Comment (randrii) [reply src/main/java/com/rybka/controller/BankController.java]: fixed

File: src/main/java/com/rybka/controller/BankController.java (hunk 2 above)
Comment (randrii) [reply src/main/java/com/rybka/controller/BankController.java]: fixed

**Pull request merged**